EFCore.FirebirdSql/Storage/Internal/Mapping/FbTypeMapper.cs

```diff
@@ -53,6 +53,7 @@ public class FbTypeMapper : RelationalTypeMapper
         private readonly FloatTypeMapping _float = new FloatTypeMapping("FLOAT");
         private readonly DoubleTypeMapping _double = new DoubleTypeMapping("DOUBLE PRECISION");
         private readonly DecimalTypeMapping _decimal = new DecimalTypeMapping($"DECIMAL({DefaultDecimalPrecision},{DefaultDecimalScale})");
+        private readonly DecimalTypeMapping _numeric = new DecimalTypeMapping($"NUMERIC({DefaultDecimalPrecision},{DefaultDecimalScale})");
```

Comment:
Where is this being used?

Reply:
If a firebird database table has a column of the type numeric. 

{ "NUMERIC", _decimal },